package.json

```diff
@@ -14,7 +14,7 @@
     "vendor"
   ],
   "types": "bson.d.ts",
-  "version": "6.7.0",
+  "version": "3.7.0",
```

Comment:
```suggestion
  "version": "6.7.0",
```

Reply:
Ah my bad, this was purely for testing, let me revert the git commits back.